cypress/e2e/inventory/actions-menu.cy.js

```diff
@@ -3,13 +3,28 @@ import InventorySearchAndFilter from '../../support/fragments/inventory/inventor
 import InventoryActions from '../../support/fragments/inventory/inventoryActions';
 import testTypes from '../../support/dictionary/testTypes';
 import devTeams from '../../support/dictionary/devTeams';
+import InventoryInstances from '../../support/fragments/inventory/inventoryInstances';
+import getRandomPostfix from '../../support/utils/stringTools';
 
-describe('ui-inventory: actions', { retries: 2 }, () => {
-  beforeEach('navigates to actions', () => {
+
+const item = {
+  instanceName: `testBulkEdit_${getRandomPostfix()}`,
+  itemBarcode: getRandomPostfix(),
+};
+describe('ui-inventory: actions', () => {
+  before('login and create test data', () => {
     cy.login(Cypress.env('diku_login'), Cypress.env('diku_password'));
+    InventoryInstances.createInstanceViaApi(item.instanceName, item.itemBarcode);
```

Comment:
Let's use this method InventoryInstances.createFolioInstanceViaApi

Reply:
createFolioInstanceViaApi mehtod uses createFolioInstanceViaApi inside
So why I should you createFolioInstanceViaApi?

Reply:
Are sure? createInstanceViaApi is not the same as createInstanceVia**API**

Reply:
![image](https://user-images.githubusercontent.com/26041116/217216985-2e005313-c6bd-48b7-b6bf-f02aace95374.png)

Reply:
So I don't understand why can not use my method as I do it in all other my tests

Reply:
disscused, solved